myriad-scheduler/src/main/resources/myriad-config-default.yml

```diff
@@ -56,6 +56,9 @@ executor:
   #The following should be used for a remotely distributed URI, hdfs assumed but other URI types valid.
   #nodeManagerUri: hdfs://namenode:port/dist/hadoop-2.7.0.tar.gz
   #path: hdfs://namenode:port/dist/myriad-executor-runnable-0.1.0.jar
+  #jvmUri: https://downloads.mycompany.com/java/jre-7u76-linux-x64.tar.gz
+  #jvmPath: jre1.7.0_76
+  #javaLibraryPath: /opt/mycompany/lib
```

Comment:
We've already got a place to set JAVA_HOME by placing it in yarnEnvironment I've used it on my system alot so am comfortable saying it works.  Is there a good reason to add code for this environment variable?

I like the idea of allowing the option to d/l Java though!